### PR TITLE
Fix for AppCloud-129

### PR DESCRIPTION
**Pull request by SabraO**

### CODE REVIEW

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java [hunk 1]
```diff
@@ -236,5 +236,5 @@
     public static final String DELETE_ALL_APP_VERSION_EVENTS =
             "Delete from AC_EVENT where version_id = (SELECT id FROM AC_VERSION WHERE hash_id=?)";
 
-	public static final String GET_TENANT_APPLICATION_COUNT = "SELECT COUNT(*) FROM AC_VERSION WHERE tenant_id = ?";
+	public static final String GET_TENANT_APPLICATION_COUNT = "SELECT COUNT(*) FROM AC_APPLICATION WHERE tenant_id = ?";
```
Comment (chalitha1989) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java]: indentation issue

Comment (chalitha1989) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java]: indentation issue

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 2]
```diff
@@ -85,6 +86,26 @@ var addApplication = function (applicationName, versionName, applicationDescript
...
 };
 
+var stopOtherVersions = function(applicationHashId, selectedRevision){
+    var application = getApplication(applicationHashId);
+    var applicationVersions = application.versions;
+    var keys = Object.keys(applicationVersions);
+
+    for(var i=0; i < keys.length; i++){
+        var key = keys[i];
+
```
Comment (isuruw) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Remove this line

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java (hunk 1 above)
Comment (SabraO) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java]: This code segment has been removed from the main repo.

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 2 above)
Comment (SabraO) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This code segment has been removed from the main repo.

**Pull request merged**